notebooks/07_triangular_data_in_kendall_shape_space.ipynb

```diff
@@ -349,6 +349,13 @@
     "The diagonal geodesics used by the Pole Ladder method to compute the parallel transport are plotted in red."
```

Comment:
<p>Mention the code within the codebase, as opposed to the pull request. I think in our case this is:https://github.com/geomstats/geomstats/blob/f9f669551faeaf8c23f347b57d1ce3970eacbe3a/geomstats/geometry/pre_shape.py#L516</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>